test/spec/app_spec.rb

```diff
@@ -832,28 +790,102 @@
     #   skip("Needs to be implemented")
     # end
 
-    it "should navigate between teams", bin4: true, quick: true do
+
+    it "should navigate between teams", bin5: true, quick: true do
```

Comment:
Just a suggestion: I think this test is way too big. We are actually testing switch teams, join team, approve user, delete user and reject user. I suggest to split into five or at least change the test title to reflect what's actually being tested here.

Reply:
Those tests are so connected that in order to avoid repetition I choose to join them and I think it worked fine.
I completely agree with the title suggestion.